src/Common/Core/Twig/Extensions/BaseTwigModifiers.php

```diff
@@ -37,7 +37,24 @@ public static function formatCurrency($string, $currency = 'EUR', $decimals = nu
             default:
         }
 
-        return $currency.'&nbsp;'.number_format((float) $string, $decimals, ',', '&nbsp;');
+        return $currency.'&nbsp;'.static::formatNumber($string, $decimals);
```

Comment:
When doing php-cs-fixer fix without --level=psr2, then spaces are getting removed between variables. If you add --level=psr2, they remain ;-P